cinema/tests/test_movie_api.py

```diff
@@ -157,3 +158,119 @@ def test_image_url_is_shown_on_movie_session_detail(self):
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
```

Comment:
The test checks for a 'movie_image' key in the response data, which might not be correct. Verify the API response structure to ensure the correct key is being checked.